### DISCUSSION TITLE
Unity and Hololens 2

### DISCUSSION THREAD

**Discussion by Darkmax** [fo-dicom/fo-dicom#1609] [Q&A]
Hi, I tried to use your plugin in unity, (I downloaded the version is in store) and it works great in editor, but when I compile this project for UWP (Universal Windows Platform) and ARM64 to use it for the  Hololens 2 it throws some erors like Dicom could not be found (are you missing directive or an assembly refference?), and I think is because the dlls or libraries that dicom is using are not compatible for this system, how can I solve this issue?

Also in another topic I notice that the version on the asset store is very old in comparison with the version that is located here on github, is possible to updated it?

**Comment by amoerie** [comment]
Hi @Darkmax none of the maintainers (AFAIK) use Unity or HoloLens, so even if we wanted to help you, we would have no means to verify that our changes work. We do welcome pull requests if you want to take the initiative.
